service/metadata/client/metadata_client.go

```diff
@@ -104,6 +104,23 @@ func (client *MetadataClient) GetUserBucketsCount(ctx context.Context, in *metat
 	return resp, nil
 }
 
+// ListExpiredBucketsBySp list buckets that are expired by specific sp
+func (client *MetadataClient) ListExpiredBucketsBySp(ctx context.Context, createAt int64, primarySpAddress string, limit int64, opts ...grpc.CallOption) ([]*metatypes.Bucket, error) {
```

Comment:
the length of 108 line may exceed 120.

Reply:
This is to follow @joeylichang 's suggestion to pass in params other than grpc structs in client. Not sure if we can reduce length of a line while still keep its functionality?